pkg/scheduler/core/util.go

```diff
@@ -79,11 +88,21 @@ func calAvailableReplicas(clusters []*clusterv1alpha1.Cluster, spec *workv1alpha
 			continue
 		}
 		for i := range res {
+			if res[i].Name != availableTargetClusters[i].Name {
+				continue
+			}
 			if res[i].Replicas == estimatorclient.UnauthenticReplica {
 				continue
 			}
-			if availableTargetClusters[i].Name == res[i].Name && availableTargetClusters[i].Replicas > res[i].Replicas {
+			// means there are already replicas calculated by higher-priority estimator
+			if estimator.GetPriority() < availableResultPriority[i] {
```

Comment:
if `SchedulerEstimator` give the result first, can we prevent calling `GeneralEstimator`?

Reply:
So I think a better solution is to sort all estimators in descending order of priority, call each estimator in turn, and only when the current estimator call fails we call the lower priority estimator.

Reply:
sorry, I didn't got your mean, what does ` give the result first` mean?

Reply:
I mean we can save time on several estimator calls？

Reply:
right, when `SchedulerEstimator` has calculated the result, do you think it is still necessary to call `GeneralEstimator`?

Reply:
you are right~

Reply:
I think the lower priority estimator is just a fallback option rather than a must-have.